src/Aspire.Hosting/ApplicationModel/DeploymentTargetAnnotation.cs

```diff
@@ -12,4 +12,12 @@ public sealed class DeploymentTargetAnnotation(IResource target) : IResourceAnno
     /// The deployment target.
     /// </summary>
     public IResource DeploymentTarget { get; } = target;
+
+    /// <summary>
+    /// Gets or sets the container registry information associated with
+    /// the deployment target, if the deployment target is an image-based environment.
+    /// </summary>
+#pragma warning disable ASPIRECOMPUTE001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
```

Comment:
Since this is public, we probably just put

```suggestion
[Experimental("ASPIRECOMPUTE001")]
```

Reply:
I wasn't sure what our stance was on experimental attributes on already released types so I opted for a suppression here.

Reply:
But it is just on the new property.